package.json

```diff
@@ -111,9 +111,9 @@
     "test:coverage": "craco test --coverage --watchAll=false",
     "test:update": "craco test --updateSnapshot",
     "eject": "react-scripts eject",
-    "lint": "npm run lint:config && npm run lint:src ",
+    "lint": "npm run lint:config && npm run lint:src",
     "lint:config": "eslint .",
```

Comment:
Probably could be the same for `config`

Reply:
OK, let's add it also there. Did not add it at first because we don't usually use `lint:config` directly. But it's not a big deal

Reply:
We use `lint` which calls it, so it would not have `--max-warnings=0` otherwise